Gemfile

```
@@ -311,6 +311,11 @@ gem 'roar', '~> 1.2.0'
 # CORS for API
 gem 'rack-cors', '~> 1.1.1'
 
+# Gmail API
+gem 'google-apis-core'
+gem 'google-apis-gmail_v1'
+gem 'googleauth'
```

```suggestion
gem 'google-apis-core', require: false
gem 'google-apis-gmail_v1', require: false
gem 'googleauth', require: false
```

As you require these dependencies in the rakefile, we can avoid loading them for the entire stack, which reduces memory consumption a bit.